### DISCUSSION TITLE
Phalcon installation issue on Xampp/Windows - Uncaught Error: Class 'Phalcon\Loader' not found

### DISCUSSION THREAD

**Discussion by wilsonsuryajaya** [phalcon/phalcon#257] [Q&A] [answered]
I have downloaded Phalcon and PSR dlls that match up with the PHP on my XAMPP environment, put the dlls in C:\xampp\php\ext folder, and edit the php.ini accordingly
- phalcon-php7.4-ts-windows2019-vc15-x64.zip from https://github.com/phalcon/cphalcon/releases/tag/v5.0.0RC3
- php_psr-1.2.0-7.4-ts-vc15-x64.zip from https://pecl.php.net/package/psr/1.2.0/windows

Both Phalcon and PSR seems to be loaded properly on php info (picture below), but if I try to run a test code I get the following error

Fatal error: **Uncaught Error: Class 'Phalcon\Loader' not found in** C:\xampp\htdocs\htdocs_PhalconSimpleRest\index.php:10 Stack trace: #0 {main} thrown in C:\xampp\htdocs\htdocs_PhalconSimpleRest\index.php on line 10

![image](https://user-images.githubusercontent.com/692107/182613272-38e28f17-09a1-492b-b153-f8952c3bd9d6.png)

Below is a portion of code that I am trying to execute

![image](https://user-images.githubusercontent.com/692107/182615157-19279a70-a919-4127-8e3f-1fed427024cb.png)

What could be the potential issue? Thanks.

**Comment by sinbadxiii** [comment, marked as answer]
@wilsonsuryajaya  In Phalcon v5, many classes have been moved to other namespaces.

https://github.com/phalcon/cphalcon/blob/master/CHANGELOG-5.0.md

Moved `Phalcon\Loader` to `Phalcon\Autoload\Loader`

**Reply by FauzanDzikry** [reply]
I still have error "Undefined type 'Phalcon\Autoload\Loader'"

**Reply by sinbadxiii** [reply]
Is version 5? Or 4?

**Reply by FauzanDzikry** [reply]
version 5

**Reply by sinbadxiii** [reply]
Could you attach this code screenshot please?

**Comment by wilsonsuryajaya** [comment]
Thank you, this solves my issue. 

I revert back to use Phalcon v4 dll [v4.1.2](https://github.com/phalcon/cphalcon/releases/tag/v4.1.2) and everything works fine.
